<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QMainWindow" name="view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>view</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
     </font>
    </property>
    <property name="text">
     <string>BLUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
     </font>
    </property>
    <property name="text">
     <string>RED</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>481</width>
      <height>37</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>32</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CS3505 SIMON GAME</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>481</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>click start</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>361</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="greenButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>121</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
     </font>
    </property>
    <property name="text">
     <string>GREEN</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
